src/components/CompareResults/RevisionRow.tsx

```diff
@@ -364,6 +365,12 @@ function RevisionRow(props: RevisionRowProps) {
             <div className='platform-container'>
               {platformIcon}
               <span>{platformShortName}</span>
+              {platformShortName === androidPlatform && (
+                <span>
+                  &nbsp;
+                  {platform.split('-')[2]}
+                </span>
+              )}
```

Comment:
It would be better to move that to `getPlatformShortName`.
The rationale is that we also want to add the platform version (especially MacOSX and Windows) and this will be even more checks, so it's better to have this well self contained in one location. Moreover there's already a block for Android there. And you'll be able to add unit tests for that ;-)

Reply:
Also when you move the code to `getPlatformShortName`, please provide one example of how the platform looks like close to this `split`, in a comment.